ocs_ci/helpers/cephx_keyrotation_helper.py

```
@@ -0,0 +1,836 @@
+                if before_state.get(pod_name) != annotation:
+                    return True
+            return False
+
+        for restarted in TimeoutSampler(timeout, sleep, _pods_restarted):
+            if restarted:
+                log.info("Pods restarted for label %s", label)
+                return self.capture_daemon_pod_state(label)
+
+        raise UnexpectedBehaviour(
+            f"Pods with label {label} did not restart within {timeout}s"
+        )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)